`clone()` declaration not overridden for `email.policy.EmailPolicy`

The `clone()` method of `email.policy.EmailPolicy` accepts keyword arguments for the properties that the class adds to `Policy`, but because there is no declaration of `clone()` [under `EmailPolicy` in `typeshed/std/email/policy.py`](https://github.com/python/typeshed/blob/de4842db26886ff5e901f11df5a7253cb5f84c96/stdlib/email/policy.pyi#L11), code like the following incorrectly fails to type-check under mypy 1.12:

```python
from email import policy

assert policy.default.utf8 is False
p = policy.default.clone(utf8=True)
assert p.utf8 is True
```

```
clone-utf8.py:4: error: Unexpected keyword argument "utf8" for "clone" of "_PolicyBase"  [call-arg]
/usr/local/Cellar/mypy/1.12.0/libexec/lib/python3.13/site-packages/mypy/typeshed/stdlib/email/_policybase.pyi:21: note: "clone" of "_PolicyBase" defined here
```

--- a/stdlib/email/policy.pyi
+++ b/stdlib/email/policy.pyi
@@ -3,6 +3,7 @@ from email._policybase import Compat32 as Compat32, Policy as Policy, _MessageFa
 from email.contentmanager import ContentManager
 from email.message import EmailMessage, Message
 from typing import Any, TypeVar, overload
+from typing_extensions import Self
 
 __all__ = ["Compat32", "compat32", "Policy", "EmailPolicy", "default", "strict", "SMTP", "HTTP"]
 
@@ -48,6 +49,22 @@ class EmailPolicy(Policy[_MessageT]):
     def header_fetch_parse(self, name: str, value: str) -> Any: ...
     def fold(self, name: str, value: str) -> Any: ...
     def fold_binary(self, name: str, value: str) -> bytes: ...
+    def clone(
+        self,
+        *,
+        max_line_length: int | None = ...,
+        linesep: str = ...,
+        cte_type: str = ...,
+        raise_on_defect: bool = ...,
+        mangle_from_: bool = ...,
+        message_factory: _MessageFactory[_MessageT] | None = ...,
+        # Added in Python 3.8.20, 3.9.20, 3.10.15, 3.11.10, 3.12.5
+        verify_generated_headers: bool = ...,
+        utf8: bool = ...,
+        refold_source: str = ...,
+        header_factory: Callable[[str, str], str] = ...,
+        content_manager: ContentManager = ...,
+    ) -> Self: ...
 
 default: EmailPolicy[EmailMessage]
 SMTP: EmailPolicy[EmailMessage]

